src/builder/Actions.sol

```diff
@@ -681,48 +612,56 @@ library Actions {
         return (quarkOperation, action);
     }
 
-    function unwrapAsset(UnwrapAsset memory unwrap, PaymentInfo.Payment memory payment, bool useQuotecall)
-        internal
-        pure
-        returns (IQuarkWallet.QuarkOperation memory, Action memory)
-    {
+    function transformToConterpartAsset(
```

Comment:
typo in counterpart

Reply:
Also similar comment here. We can just name it `wrapOrUnwrapAsset` to be consistent with the new action context name